lib/src/main/java/io/github/davidallison/android/sdk/penremote/AirMotionEvent.kt

```diff
@@ -37,10 +37,13 @@ package io.github.davidallison.android.sdk.penremote
  * https://developer.samsung.com/galaxy-spen-remote/api-reference/com/samsung/android/sdk/penremote/AirMotionEvent.html
  */
 class AirMotionEvent(event: SPenEvent) {
+    val deltaX: Float
+    val deltaY: Float
+    val timeStamp: Long
+
     init {
-        TODO("set variables using the SpenEvent")
+        timeStamp = event.timeStamp
+        deltaX = event.values[1]
+        deltaY = event.values[2]
     }
```

Comment:
ditto: Kotlin allows for variables to be defined inline, rather than in an init block